ingest/importer/conversion/metadata_entity.py

```diff
@@ -86,11 +86,18 @@ def _do_add_links(link_map, link_entity_type, new_links):
             link_map[link_entity_type] = existent_links
         existent_links.extend(new_links)
 
-    def retain_content_fields(self, *fields):
+    def retain_intersecting_fields(self, *fields):
         for key in self._content.keys():
             if key not in fields:
                 self._content.remove_field(key)
 
+    def list_non_intersecting_fields(self, fields):
+        non_intersecting_fields = []
+        for key in self._content.keys():
+            if key not in fields:
+                non_intersecting_fields.append({'key': key, 'value': self._content[key]})
+        return non_intersecting_fields
+
```

Comment:
I think this can be shortened into something like, 

    [{'key': key, 'value': self._content[key]} for key in self._content_keys() if key not in fields] 

Also, `list_non_intersecting_fields` doesn't sound very descriptive to me. If the intent is to filter out the given fields, wouldn't it be more appropriate to call it using something like `filter_out(fields)` or `list_fields(excluding=excluded_fields)`.